pages/docs/config/apps.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Add Apps"
+title: "Apps"
 date: "2020-08-01"
-section: "guides"
+section: "config"
```

Comment:
Do the `sections` mater any more since the nav is built up manually?

Reply:
They are being used by the breadcrumbs. I'm currently working a story to rework the breadcrumbs and that should enable me to remove all of these.